lib/domain/repository/chat.dart

```diff
@@ -116,16 +116,16 @@ abstract class AbstractChatRepository {
   Future<void> readChat(ChatId chatId, ChatItemId untilId);
 
   /// Edits the specified [ChatMessage] posted by the authenticated [MyUser].
-  Future<void> editChatMessageText(ChatItemId itemId, ChatMessageText? text);
+  Future<void> editChatMessageText(Rx<ChatItem> item, ChatMessageText? text);
```

Comment:
@andrigel @SleepySquash why do we change repository interface this way? It seems unobvious why we decide to update stored `ChatItem` depending on some arbitrary `Rx<ChatItem>` passed externally, because we cannot guarantee here that the passed `Rx<ChatItem>` has the same props as the one stored in `Repository`. We may just fetch the stored `ChatItem` inside the method, leaving its signature as simple as it was. As we do for `removeChatMember(ChatId chatId, UserId userId)` method, for example.